connector-dev.md

```diff
@@ -0,0 +1,70 @@
+# Connector Development Standards
+
+MeltanoLabs is primarily a place for well written SDK based connectors.
+As part of that there's an expected level of quality that each connector should strive to meet.
+Many times the connectors in MeltanoLabs are built from scratch as a brand new connector or rebuilt to replace a legacy one.
+With that in mind this is a compiled list of the development best practices and standards for adding a connector to MeltanoLabs.
+This can also act as a guide for others outside the MeltanoLabs organization to understand the best approach to building a high quality connector.
+
+## Tips For Building From Scratch
+
+### Start With the cookiecutter Templates
+
+When building from scratch always use the cookiecutter to make sure the newest features and recommended scaffold is present.
+This is compared to starting with a clone of an existing SDK repo and modifying it for the new source, which could miss out on some improvements.
+
+### Use the Latest SDK Version
+
+Similarly you should always start development of a new connector using the latest SDK version.
+The SDK has had major improvements over time so making sure you're taking advantage of the latest features and not accidentally reinventing the wheel is important.
+
+### Naming Convention
+
+Generally when deciding the name for a connector we use the `tap-<source>` naming convention.
+- Start with `tap-` or `target-`
+- Use dashes over underscores
+- Avoid extras like `tap-<source>-sdk`, `meltano-tap-<source>`, `tap-<source>-new`, `target-<destination>-loader`, etc.
+- If rebuilding an existing connector it's usually best to keep the same naming convention as existing legacy connectors for clarity within the community.
+For example prefer `tap-facebook` to `tap-meta` (Facebook's new name) because theres a long history of taps named `tap-facebook` and will be more clear.
```

Comment:
```suggestion
For example, prefer `tap-facebook` to `tap-meta` (Facebook's new name) because there's a long history of taps named `tap-facebook` and will be more clear.
```